File: src/main/java/com/hoi4utils/clausewitz/exceptions/NotPermittedInScopeException.java
package com.hoi4utils.clausewitz.exceptions;

public class NotPermittedInScopeException extends Exception {
	public NotPermittedInScopeException() {
		super();
	}

	public NotPermittedInScopeException(String message) {
		super(message);
	}

	public NotPermittedInScopeException(String message, Throwable cause) {
		super(message, cause);
	}

	public NotPermittedInScopeException(Throwable cause) {
		super(cause);
	}
}

File: src/main/scala/com/hoi4utils/exceptions/NotPermittedInScopeException.scala
package com.hoi4utils.exceptions

class NotPermittedInScopeException(message: String, cause: Throwable) extends Exception(message, cause) {

  def this(cause: Throwable) =
    this(null, cause)
    
  def this(message: String) =
    this(message, null)
}
